style.css

```diff
@@ -0,0 +1,47 @@
+body {
+	font-family: sans-serif;
+	align: center;
+}
+
+h1 {
+	background: white;
+	text-align: center;
+	padding: 15px;
+}
+
+p {
+	color: black;
+	margin-bottom: 10px;
+	font-size: 12px
+}
+
+p.main {
+	margin-bottom: 15px;
+}
+
+code {
+	color: black;
+	background: white;
+	padding: 2px 5px;
+	border: 1px solid black;
+}
+
+img {
+	padding: 50px;
+	align:center;
```

Comment:
:hammer: Did you mean _text_-align? Looks like you want the image to be in the middle of the mage. You would have to do something like this:

HTML:
```html
<div class="center">
  <img src="somefile.png>
</div>
```

CSS:
```css
.center {
  text-align: center;
}
```

OR... we could simply change the display of the image and do a trick with the margin:
HTML:
```html
<img class="centered-image" src="someimage.png">
```

CSS:
```css
.centered-image {
  display: block; /* this turns the image to a block element, taking up the whole line */
  margin: 0 auto; /* this will put equal margin on the left at right sides, effectively centering the image */
}
```